include/nuttx/sched_note.h

```diff
@@ -99,6 +99,15 @@
   memset((s), 0, sizeof(struct note_filter_irq_s))
 #endif
 
+/* Note dump module tag definitions */
+
+#ifdef CONFIG_SCHED_INSTRUMENTATION_DUMP
+#define NOTE_MODULE(a, b, c, d) \
```

Comment:
```suggestion
#  define NOTE_MODULE(a, b, c, d) \
```

Reply:
Where `NOTE_MODULE` macro is used?

Reply:
@pkarashchenko 
#define TEST_MODULE NOTE_MODULE('t', 'e', 's', 't')
sched_note_dump(TEST_MODULE, event, data, len);

#define AUDIO_MODULE NOTE_MODULE('a', 'u', 'd', 'i')
sched_note_dump(AUDIO_MODULE, event, data, len);
or
sched_note_dump(NOTE_MODULE('a', 'u', 'd', 'i'), event, data, len);

Reply:
So that is from `apps` repo right?

Reply:
yes, i will have a new PR for "NOTE_MODULE" in apps.